### PR TITLE
fluid-framework README

### PR DESCRIPTION
**Pull request by ChumpChief**
Fleshing out the README for the fluid-framework, primarily linking off to other doc pages.

### CODE REVIEW

File: packages/framework/fluid-framework/README.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # fluid-framework
 
-The main entry point into Fluid Framework client packages
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a corresponding service client package, such as the `@fluidframework/azure-client` package.
+
+## Contents
+
+The `fluid-framework` package consists primarily of two portions:  the `IFluidContainer` and a selection of DDS packages.
```
Comment (tylerbutler) [inline packages/framework/fluid-framework/README.md:7]:
```suggestion
The `fluid-framework` package consists primarily of two portions:  the `IFluidContainer` and a selection of distributed data structures (DDSes).
```

File: packages/framework/fluid-framework/README.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+You'll use one or more DDS data structures in your container to model your collaborative data.  The `fluid-framework` package comes with three data structures that cover a broad range of scenarios:
+1. **[SharedMap](https://fluidframework.com/docs/apis/map/sharedmap/)**, a map-like data structure for storing key/value pair data
+2. **[SharedDirectory](https://fluidframework.com/docs/apis/map/shareddirectory/)**, a map-like data structure with ability to organize keys into subdirectories
+3. **[SharedString](https://fluidframework.com/docs/apis/sequence/sharedstring/)**, a data structure for string data
+
+## Tutorial
+
+Check out the Hello World tutorial using the `fluid-framework` package [here](https://fluidframework.com/docs/start/tutorial/).
 
 See [GitHub](https://github.com/microsoft/FluidFramework) for more details on the Fluid Framework and packages within.
```
Comment (tylerbutler) [inline packages/framework/fluid-framework/README.md:24]:
```suggestion
See [GitHub](https://github.com/microsoft/FluidFramework) for more details on the Fluid Framework and packages within.

<!-- Links -->
[IFluidContainer]: https://fluidframework.com/docs/apis/fluid-static/ifluidcontainer/
[SharedMap]: https://fluidframework.com/docs/apis/map/sharedmap/
[SharedDirectory]: https://fluidframework.com/docs/apis/map/shareddirectory/
[SharedString]: https://fluidframework.com/docs/apis/sequence/sharedstring/
```

Moving the links to the bottom as reference links makes it easier to re-use the readme elsewhere.

File: packages/framework/fluid-framework/README.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
-The main entry point into Fluid Framework client packages
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a corresponding service client package, such as the `@fluidframework/azure-client` package.
+
+## Contents
+
+The `fluid-framework` package consists primarily of two portions:  the `IFluidContainer` and a selection of DDS packages.
+
+### IFluidContainer
+
+The **[IFluidContainer](https://fluidframework.com/docs/apis/fluid-static/ifluidcontainer/)** interface is the one of the types returned by calls to `createContainer()` and `getContainer()` on the service clients such as `AzureClient`.  It includes functionality to retrieve the Fluid data contained within, as well as to inspect the state of the collaboration session connection.
```
Comment (tylerbutler) [inline packages/framework/fluid-framework/README.md:11]:
```suggestion
The **[IFluidContainer][]** interface is the one of the types returned by calls to `createContainer()` and `getContainer()` on the service clients such as `AzureClient`.  It includes functionality to retrieve the Fluid data contained within, as well as to inspect the state of the collaboration session connection.
```

File: packages/framework/fluid-framework/README.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+
+### DDS packages
+
+You'll use one or more DDS data structures in your container to model your collaborative data.  The `fluid-framework` package comes with three data structures that cover a broad range of scenarios:
+1. **[SharedMap](https://fluidframework.com/docs/apis/map/sharedmap/)**, a map-like data structure for storing key/value pair data
+2. **[SharedDirectory](https://fluidframework.com/docs/apis/map/shareddirectory/)**, a map-like data structure with ability to organize keys into subdirectories
+3. **[SharedString](https://fluidframework.com/docs/apis/sequence/sharedstring/)**, a data structure for string data
```
Comment (tylerbutler) [inline packages/framework/fluid-framework/README.md:18]:
```suggestion
1. **[SharedMap][]**, a map-like data structure for storing key/value pair data
2. **[SharedDirectory][]**, a map-like data structure with ability to organize keys into subdirectories
3. **[SharedString][]**, a data structure for string data
```

**Review by tylerbutler — APPROVED** [review]

**Comment by tylerbutler** [comment]
@skylerjokiel Do you think we should include a list of the core packages that this package encompasses?

File: packages/framework/fluid-framework/README.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
 # fluid-framework
 
-The main entry point into Fluid Framework client packages
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a corresponding service client package, such as the `@fluidframework/azure-client` package.
```
Comment (skylerjokiel) [inline packages/framework/fluid-framework/README.md:3]:
```suggestion
The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a corresponding service client package (ex. `@fluidframework/azure-client` & `@fluidframework/tinylicious-client` ).
```

**Review by skylerjokiel — APPROVED** [review]
🍁

**Pull request merged**